test/general/config.js

```diff
@@ -136,11 +136,11 @@ vAFM3jjrAQDgJPXsv8PqCrLGDuMa/2r6SgzYd03aw/xt1WM6hgUvhQD+J54Z
       const { key, revocationCertificate } = await openpgp.generateKey({ userIDs });
 
       const opt = { key };
-      const { privateKeyArmored: revKeyArmored } = await openpgp.revokeKey(opt);
+      const revKeyArmored = await openpgp.revokeKey(opt).then(revKey => revKey.armor(opt.config));
```

Comment:
do you prefer this broken into multiple variable/awaits?

```suggestion
      const revKey = await openpgp.revokeKey(opt);
      const revKeyArmored = await revKey.armor(opt.config);
```